rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -23,21 +23,60 @@ components:
       bearerFormat: JWT    # optional, arbitrary value for documentation purposes
 security:
 - bearerAuth: []         # use the same name as above
-paths:
 
+paths:
 # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 # RECORDINGS
 # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
   /user_profiles/{user_profile_id}/recordings:
     post:
       description: |
-        * **Available since:** NOT YET AVAILABLE
-      summary: # TODO
+        * **Available since:** 2019-11 (BPLAT-12079)
+      summary: Create a recording of a given broadcast slot for a given user profile
       tags:
         - Recordings
       operationId: createRecordedSlot
-      # TODO
+      parameters:
+        - in: path
+          name: user_profile_id
+          required: true
+          description: The identity of the user profile to create recording groups for.
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/UserRecordings/RecordingEntity.yml#/CreateRecordingRequestEntity"
+
+      responses:
+        201:
+          description: "Successful response on created resource"
+          content:
+            application/json:
+              schema:
+                $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/UserRecordings/RecordingEntity.yml#/RecordingEntity"
```

Comment:
Somehow, the ref does not work when previewing online. But I can download it manually. Don't know if something is wrong, but it might be worth another look.